### ISSUE TITLE
404 Not Found when request api key

### ISSUE THREAD

**Issue by YiruiGao** [lukaswhite/weboftrust#2]
Hi, Lukas, I'm try to request an api key, but when i click the link, it always jump to a 404 not found page, can you help me?

**Comment by lukaswhite** [comment]
You're right, that link doesn't work any more, and neither does the link [from the docs](https://www.mywot.com/wiki/index.php/API).

I've contacted their support team to find out what the new link should be, assuming the API is still active.
